### ISSUE TITLE
Parsing ![]() is buggy

### ISSUE THREAD

**Issue by jackens** [markedjs/marked#1456]
**Describe the bug**
Parsing `![]()`  is buggy

**To Reproduce**

```md
![](img1.svg) (or ![](img2.svg))
```

generates

```html
<p><img src="img1.svg" alt="" title="or ![](img2.svg">)</p>
```

(conf. <https://marked.js.org/demo/?text=![](img1.svg)%20(or%20![](img2.svg))>)

**Expected behavior**
```html
<p><img src="img1.svg" alt=""> (or <img src="img2.svg" alt="">)</p>
```

**Comment by UziTech** [comment]
This should be fixed in #1453

[demo](https://marked.js.org/demo/?outputType=html&text=!%5B%5D(https%3A%2F%2Fexample.com%2F1)%20(!%5B%5D(https%3A%2F%2Fexample.com%2F2))%20!%5B%5D(https%3A%2F%2Fexample.com%2F3)&options=%7B%0A%20%22baseUrl%22%3A%20null%2C%0A%20%22breaks%22%3A%20false%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22headerIds%22%3A%20true%2C%0A%20%22headerPrefix%22%3A%20%22%22%2C%0A%20%22highlight%22%3A%20null%2C%0A%20%22langPrefix%22%3A%20%22language-%22%2C%0A%20%22mangle%22%3A%20true%2C%0A%20%22pedantic%22%3A%20false%2C%0A%20%22sanitize%22%3A%20false%2C%0A%20%22sanitizer%22%3A%20null%2C%0A%20%22silent%22%3A%20false%2C%0A%20%22smartLists%22%3A%20false%2C%0A%20%22smartypants%22%3A%20false%2C%0A%20%22tables%22%3A%20true%2C%0A%20%22xhtml%22%3A%20false%0A%7D&version=pr%3A1435)

**Comment by UziTech** [comment]
I'm not sure why the demo's master isn't working since #1435 was already merged, but this is already fixed in master.

**Comment by UziTech** [comment]
demo master should be fixed in #1457

**Comment by UziTech** [comment]
This should be fixed in v0.6.2 [demo](https://marked.js.org/demo/?outputType=html&text=!%5B%5D(img1.svg)%20(or%20!%5B%5D(img2.svg))&options=%7B%0A%20%22baseUrl%22%3A%20null%2C%0A%20%22breaks%22%3A%20false%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22headerIds%22%3A%20true%2C%0A%20%22headerPrefix%22%3A%20%22%22%2C%0A%20%22highlight%22%3A%20null%2C%0A%20%22langPrefix%22%3A%20%22language-%22%2C%0A%20%22mangle%22%3A%20true%2C%0A%20%22pedantic%22%3A%20false%2C%0A%20%22sanitize%22%3A%20false%2C%0A%20%22sanitizer%22%3A%20null%2C%0A%20%22silent%22%3A%20false%2C%0A%20%22smartLists%22%3A%20false%2C%0A%20%22smartypants%22%3A%20false%2C%0A%20%22tables%22%3A%20true%2C%0A%20%22xhtml%22%3A%20false%0A%7D&version=0.6.2)
